src/ds/features/auto-exposure-gain-limit-feature.h

```diff
@@ -0,0 +1,27 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <src/feature-interface.h>
+
+#include <memory>
```

Comment:
Why is this needed?

Reply:
Because I'm using std::shared_ptr, it isn't compiling without it.